### DISCUSSION TITLE
https://plugin-comment2.vuejs.press/zh/guide/giscus.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#553] [Announcements]
# https://plugin-comment2.vuejs.press/zh/guide/giscus.html

Giscus 是一个基于 GitHub Discussion 的评论系统，启用简便。 准备工作 1. 你需要创建一个公开仓库，并开启评论区，以作为评论存放的地点 1. 你需要安装 Giscus App (https://github.com/apps/giscus)，使其有权限访问对应仓库。 1. 在完成以上步骤后，请前往 Giscus 页面 (htt...

https://plugin-comment2.vuejs.press/zh/guide/giscus.html

**Comment by dablelv** [comment]
您好，请问 VuePress 1.x 的版本的默认主题可以按照本文的指引集成 Giscus 吗？

**Reply by Mister-Hope** [reply]
不能

**Comment by lingdu990130556** [comment]
这里的中文是怎么配置的啊

**Reply by Mister-Hope** [reply]
自动应用页面语言

**Comment by lingdu990130556** [comment]
这里默认是最早，就会不显示评论。能默认显示最新吗
![image](https://github.com/vuepress-theme-hope/giscus-discussions/assets/76582458/8260dd4e-0a20-43f2-b898-ce8166d8d8a3)
![image](https://github.com/vuepress-theme-hope/giscus-discussions/assets/76582458/38ac27eb-b4f8-4895-a258-a966992e8f63)

**Comment by lingdu990130556** [comment]
如果项目部署在码云上出现这种情况是网络的问题吗，部署在GitHub上可以，本地运行也行，
![image](https://github.com/vuepress-theme-hope/giscus-discussions/assets/76582458/d2562b14-616b-4c7a-b1ef-78af5704fe53)

**Reply by lingdu990130556** [reply]
知道原因了，地址栏域名不能有下划线。。。改域名就好了

**Reply by ylnlh** [reply]
我的没有线也不行，是为什么

**Reply by Mister-Hope** [reply]
是最新版本么？

**Comment by zhanglinweigithub** [comment]
您好，请问一下，vuepress-theme-hope主题使用了Giscus评论功能，如何添加浏览量？还是只有Waline才支持显示页面浏览量？
